Flowauth docs update (#5886)
Flowminder/FlowKit: docs/source/administrator/deployment.md

| RESET_FLOWAUTH_DB | Set to true to reset the database | |
| FLOWAUTH_CACHE_BACKEND | Backend to use for two factor auth last used key cache | Defaults to 'file'. May be set to 'memory' if deploying a single instance on only one CPU, or to 'redis' for larger deployments |

##### Migration

> If updating Flowauth from v 1.17 to v 1.18.1, you will need to run the Alembic migration to upgrade Flowauth's backing database without losing your user data. *It is recommended that you take a backup of your flowauth db before migration*

Reviewer: ```suggestion
If updating an existing installation of FlowAuth to v1.18.1 or newer, you will need to run the Alembic migration to upgrade Flowauth's backing database without losing your user data, and then re-upload the API specifications for any existing servers in the new 1.18.1+ format. *It is recommended that you take a backup of your flowauth db before migration*
```

Once FlowAPI has started, it can be added to FlowAuth so that users can generate tokens for it. You should be able to download the API specification from `https://<flowapi_host>:<flowapi_port>/api/0/spec/openapi.json`. You can then use the spec file to add the server to FlowAuth by navigating to Servers, and clicking the new server button.

> After uploading the specification, you can configure the maximum lifetime and final expiry date of tokens issued by that server. You should then create some new roles; it is recommended that you create a role for analysis staff that includes at least the `get_results`, `get_available_dates` and `run_query` scopes, along with the appropriate geographic levels. If you are unsure, tick all boxes.

Reviewer: ```suggestion
After uploading the specification, you can configure the maximum lifetime and final expiry date of tokens issued by that server. You should then create some new roles; we recommend that you create a role for analysis staff that includes at least the `get_results`, `get_available_dates` and `run_query` scopes, along with the appropriate geographic levels. If you are unsure, tick all boxes.
```
